wcfsetup/install/files/lib/system/package/PackageUpdateUnauthorizedException.class.php

```diff
@@ -61,13 +61,25 @@ public function getRenderedTemplate()
         $serverReply = $this->request->getReply();
 
         $authInsufficient = (($serverReply['httpHeaders']['wcf-update-server-auth'][0] ?? '') === 'unauthorized');
-        if (ENABLE_ENTERPRISE_MODE && $authInsufficient && !empty($this->packageUpdateVersion['pluginStoreFileID'])) {
-            WCF::getTPL()->assign([
-                'packageName' => $this->packageUpdateVersion['packageName'],
-                'pluginStoreFileID' => $this->packageUpdateVersion['pluginStoreFileID'],
-            ]);
+        if ($authInsufficient && !empty($this->packageUpdateVersion['pluginStoreFileID'])) {
+            $hasOnlyTrustedServers = true;
+            if (!ENABLE_ENTERPRISE_MODE) {
```

Comment:
I believe this check is unnecessary and can be removed to keep the control flow simpler.